cmd/subctl/join.go

```diff
@@ -83,6 +83,8 @@ func addJoinFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&joinFlags.PreferredServer, "preferred-server", false,
 		"enable this cluster as a preferred server for dataplane connections")
 
+	cmd.Flags().BoolVar(&joinFlags.AirGappedDeployment, "air-gapped", false,
+		"specifies that the cluster is in an air-gapped environment")
```

Comment:
Add a little more info?

```suggestion
		"specifies that the cluster is in an air-gapped environment where external access is not allowed")
```

Reply:
I'm fine if you want me to update the description. But normally an "air-gapped" env' itself means that it does not have access to external network.

https://www.techtarget.com/whatis/definition/air-gapping#:~:text=An%20air%20gap%20is%20a,other%20computers%20or%20network%20devices.

https://en.wikipedia.org/wiki/Air_gap_(networking)

Reply:
OK - we can omit the extra info I suggested - I'm fine with whatever you think.